apache-rat-core/src/main/java/org/apache/rat/OptionCollection.java

```diff
@@ -581,7 +560,7 @@ static Optional<IOFileFilter> parseExclusions(final Log log, final List<String>
         if (ignoredLines > 0) {
             log.info("Ignored " + ignoredLines + " lines in your exclusion files as comments or empty lines.");
         }
-        return orFilter.getFileFilters().isEmpty() ? Optional.empty() : Optional.of(orFilter);
+        return orFilter.getFileFilters().isEmpty() ? Optional.empty() : Optional.of(new NotFileFilter(orFilter));
```

Comment:
Somehow this line is odd to read .....
OrFilter does not look similar to NotFileFilter to me.

Has there been a semantic change in newer apache-commons-versions? or should the variable be renamed to notFileFilter here?

Or does the change invert the logics of the CLI option (then we should add a big note to the webpage ;))